webapp/views/App/views/Users/UserEdit/UserEdit.js

```diff
@@ -38,79 +39,100 @@ const UserEdit = () => {
   const i18n = useI18n()
   const validation = User.getValidation(userToUpdate)
 
+  const titleItems = [
```

Comment:
you can simplify this with a map! ['mr', 'ms', 'preferNotToSay'].map(key => ({key, value: i18n.t(`user.title.${key}`)})
is itemKey required?